v3.20/ros/jazzy/ros-jazzy-rclcpp/wait-loading-shared-library.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/src/rclcpp/typesupport_helpers.cpp b/src/rclcpp/typesupport_helpers.cpp
+index 04dd8c4c..6a35f304 100644
+--- a/src/rclcpp/typesupport_helpers.cpp
++++ b/src/rclcpp/typesupport_helpers.cpp
+@@ -122,6 +122,11 @@ const void * get_typesupport_handle_impl(
+       package_name + "__" + middle_module + "__" + type_name;
+     const void * (* get_ts)() = nullptr;
+     // This will throw runtime_error if the symbol was not found.
++    for (size_t i=0; i < 1000000; ++i) {
++      if (library.has_symbol(symbol_name)) {
++        break;
++      }
++    }
```

Comment:
I don't know the exact reason, but `library.get_symbol()` fails if it's just after the library is loaded.
Adding a delay before `library.get_symbol()` solves that problem.
`1000000`  might not be an appropriate value.

Reply:
It would be better to avoid using 100% CPU. Is this function repeatedly called?